oqsprov/oqs_prov.h

```diff
@@ -167,15 +168,17 @@ struct oqsx_key_st {
      */
     size_t privkeylen;
     size_t pubkeylen;
+    size_t *privkeylen_cmp;
+    size_t *pubkeylen_cmp;
     size_t bit_security;
     char *tls_name;
 #ifndef OQS_PROVIDER_NOATOMIC
     _Atomic
 #endif
         int references;
 
-    /* point to actual priv key material -- classic key, if present, first
-     * i.e., OQS key always at comp_*key[numkeys-1]
+    /* point to actual priv key material -- classic key, if present, first,
+     * unless is composite i.e., OQS key always at comp_*key[numkeys-1]
```

Comment:
don't quite understand this: Where's the key material for composite, then?

Reply:
its in there, but the order is not the same, the hybrid will have Classic then PQ, Composite its the opposite, PQ then classic.
This "unless is composite" is referring to the point that the classic key will not be the first.

Reply:
Ah, OK. Then the subsequent text no longer is accurate, though: OQS key is at index 0 for composite, then, right?

Reply:
yes, the i.e. is only valid for hybrids.
I'll update the comment with more accurate text.